theorems/T000663.md

```diff
@@ -0,0 +1,14 @@
+---
+uid: T000663
+if:
+  and:
+  - P000019: true
+  - P000083: true
+then:
+  P000016: true
+refs:
+- doi: 10.1016/c2009-0-12309-7
+  name: Handbook of Set-Theoretic Topology
+---
+
+See {{doi:10.1016/c2009-0-12309-7}}, Chapter 9, 9.2 and 4.1.
```

Comment:
I'm trusting [Ulli's comment here](https://math.stackexchange.com/questions/5008287/is-omega-1-meta-lindel%C3%B6f?noredirect=1#comment10738525_5008287) - the handbook is in my office and I'm not there right now.

Reply:
![Zrzut ekranu 2024-12-08 172629](https://github.com/user-attachments/assets/e2f4b41c-18ad-4bdf-8800-34e8534598a1)

![image](https://github.com/user-attachments/assets/3343f064-ff60-4f2c-a532-ba4e8ee8c551)

Those are the theorems (one of which is a diagram) that Ulli is referencing. @StevenClontz

Reply:
![image](https://github.com/user-attachments/assets/58b3a71a-e85b-45e5-8b4b-8e6b3fdf7661)

There is also this comment at the beginning of section 9 of chapter 9